package.json

```diff
@@ -17,6 +18,7 @@
     "prebuild": "babel --copy-files ./components --out-dir lib --ignore *.scss,*.md &",
     "styleguide-build": "styleguidist build",
     "styleguide-server": "styleguidist server",
+    "styleguide:dev": "npm run styleguide-build && npm run styleguide-server",
```

Comment:
👍

Reply:
I'd like to namespace everything in the same way. I really like the combination of the other two, so let's try to make them all the same? Change the `-` to `:` in the other two?